.github/workflows/pull_request.yml

```
@@ -135,7 +135,7 @@ jobs:
           python-version: ${{ env.DESIRED_PYTHON_VERSION }}
 
       - name: Install python dependencies
-        run: pip install build
+        run: pip install build==1.5.0 --only-binary :all:
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)